src/main/java/com/thealgorithms/sorts/ExchangeSort.java

```diff
@@ -0,0 +1,34 @@
+package com.thealgorithms.sorts;
+
+import java.util.Arrays;
+
+public class ExchangeSort {
```

Comment:
`ExchangeSort` should implement [`SortAlgorithm`](https://github.com/TheAlgorithms/Java/blob/a216cb8a59ad04b3cadcc156e69dea41f3d1b465/src/main/java/com/thealgorithms/sorts/SortAlgorithm.java#L11). Please have a look at [`BubbleSort`](https://github.com/TheAlgorithms/Java/blob/a216cb8a59ad04b3cadcc156e69dea41f3d1b465/src/main/java/com/thealgorithms/sorts/BubbleSort.java#L10).